.travis.yml

```diff
@@ -33,20 +33,20 @@ jobs:
   include:
     - stage: Test
       env: LOWEST_DEPENDENCIES
-      php: 7.1
+      php: 7.2
       install:
         - travis_retry composer update --no-interaction --no-suggest --prefer-dist --prefer-lowest
 
     - stage: Test
       env: DEV_DEPENDENCIES
-      php: 7.2
+      php: 7.4snapshot
```

Comment:
The `php-7.4` version is available now.

It should use `php-7.4` version instead.